test/lib/compile-nools-rules.spec.js

```diff
@@ -103,4 +103,19 @@ describe('compile nools-rules', () => {
         expect(err.message).to.include('tasks.js');
       });
   });
+
+  it('comments and urls in fields yields exception', () => {
+    const mocks = genMocks();
+    mocks.fs.exists
+      .withArgs('/rules.nools.js').returns(false)
+      .withArgs('/targets.js').returns(true)
+      .withArgs('/tasks.js').returns(true);
+    mocks.fs.read.withArgs('/rules.nools.js').returns('define Task {_id: null, title: "http://foo.bar.com"}');
```

Comment:
There's no point mocking `/rules.nools.js` because you've just told it that it doesn't exist (fs.exists returns false on line 110).

What I think you need to do is mock `mocks.pack` to return 'http://foo.bar.com'.